.github/workflows/ui-tests/cypress/integration/create_spec.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+describe('User can navigate to create account screen', function() {
+    it('from login', function() {
+        cy.visit('/login')
+        cy.get('.btn-create-account').click()
```

Comment:
nit: A cleaner approach might be to add an `id` to the signup button and use that in the selector rather than the `class`. This would prevent tests from breaking on style changes and keeps it consistent with the rest of the `cy.get` usage in these tests. [Cypress best practices](https://docs.cypress.io/guides/references/best-practices.html) recommends avoiding the use of class selectors as well. WYDT?

Reply:
great point, totally agree!